docs/controller-skills/military.md

```diff
@@ -55,11 +55,13 @@ Some Restricted Areas will be activated by NOTAM, most commonly for use by RAAF
 </figure>
 
 ### Aircraft operating inside Restricted Area
-Military aircraft are permitted to operate within their designated Restricted Area as per the relevant NOTAM. Even if this Restricted Area is inside your airspace, you have no control authority over it.
+Military aircraft are permitted to operate within their designated Restricted Area as per the relevant NOTAM. Even if this Restricted Area is inside your airspace, you have no control authority over it. If a AWEC aircraft is up and certified, you transfer all traffic to that aircraft. The AWEC aircraft will control up to the lateral boundries of the restricted airspace. AWEC aircraft will hand off departing traffic back to your control authority at 10NMs from the boundary.
 
 !!! example
     **ARA**: "DRGN379, shortly entering R604 Restricted Area, identification and control services terminated, frequency change approved. Contact me prior to leaving for airways clearance"  
     **DRGN379**: "Wilco, DRGN379"
+    **ARA**: "ASY101, shortly entering M550 Restricted Area, contact WGTL47 on 130.500, frequency change approved"
```

Comment:
Sorry I'm a little bit confused by this example, could you please shed some light on it for me? I don't think ATC would instruct someone to contact an aircraft on a frequency, as far as I'm aware.

Reply:
Since the AEWC own the RA, controllers should transfer them to the controller in the AWEC as its their airspace

Reply:
As per https://vatpac-home.s3.ap-southeast-2.amazonaws.com/LOA_RAA_Fv_8dbadf82fa.pdf

Reply:
> As per https://vatpac-home.s3.ap-southeast-2.amazonaws.com/LOA_RAA_Fv_8dbadf82fa.pdf

Interesting, some good stuff in this document. I've never seen it before. Do you know where it's published? I may have to have some conversations up the chain, as we can't reference a document that's not published anywhere

Reply:
Its an old loa from 2019 so dont know where current loas are  but I found the doc in the RAAFv discord.

Reply:
> Its an old loa from 2019 so dont know where current loas are  but I found the doc in the RAAFv discord.

Okay thanks for the info. Am just having some discussions with other members of the ATS team regarding how we approach this one. As far as we can tell, this is an inactive policy, and there's no active policy to replace it. So I'll get back to you shortly with how that impacts this part of your work.

Thanks again for your work

Reply:
Yeah no worries, its still an unofficial thing that occurs when we run an event.

Reply:
We've decided to remove the references to the AWEC stuff, primarily because we have no active LoA or policy governing it. In the future, we'll look to add it in if we do get an active LoA up and running. Thanks again for your work

Reply:
Yeah I get that no worries.